notebooks/aira_evaluation_tutorial.ipynb

```diff
@@ -0,0 +1,679 @@
+        "# Example:\n",
+        "# !nat eval --config_file \"{config_path}\" --dataset \"{dataset_path}\n",
+        "\n",
+        "\n",
+        "import os\n",
+        "\n",
+        "# Hosted Nemotron Backend (for reasoning)\n",
+        "os.environ[\"NEMOTRON_LLM_BASE_URL\"]  = \"http://nim-llm-ms:8000/v1\"\n",
+        "# RAG server for generate_summary / artifact_qa\n",
+        "os.environ[\"RAG_SERVER_URL\"]         = \"http://rag-server:8081/v1\"\n",
+        "\n",
+        "os.environ[\"EVAL_LLM_BASE_URL\"]         = \"https://integrate.api.nvidia.com/v1\"\n",
+        "\n",
+        "\n",
+        "\n",
+        "# NVIDIA hosted models (leave as-is if you still want integrate.api)\n",
+        "# os.environ[\"RAGAS_LLM_BASE_URL\"]    = \"https://integrate.api.nvidia.com/v1\"\n",
+        "\n",
+        "\n",
+        "print(\"Environment overrides set\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# This will run the evaluation with the default config file that exists in the configs folder called eval_config.yml\n",
+        "# The workflow & metrics output will be in the notebooks/.tmp/aiq_aira_similarity folder.\n",
+        "from pathlib import Path\n",
+        "\n",
+        "try:\n",
+        "    notebook_dir = Path(globals()['_dh'])\n",
+        "except Exception:\n",
+        "    notebook_dir = Path.cwd()\n",
+        "\n",
+        "\n",
+        "# Project root is the parent of notebooks/ in this repo layout\n",
+        "project_root = notebook_dir.parent\n",
+        "\n",
+        "# Absolute path to the other config file\n",
+        "eval_config_path = (project_root / \"configs\" / \"eval_config.yml\").resolve()\n",
+        "\n",
+        "# Fail early if missing\n",
+        "assert eval_config_path.is_file(), f\"Config file not found at: {eval_config_path}\"\n",
+        "\n",
+        "# Option A: Run nat with our evaluation harness if you set the dataset path in the config file\n",
+        "!nat eval --config_file \"{eval_config_path}\"\n",
+        "\n",
+        "# Option B: Run nat with our evaluation harness if you want to set the dataset path in the command line (uncomment this line and comment the line above)\n",
+        "# !nat eval --config_file \"{eval_config_path}\" --dataset \"{dataset_path}\"\n",
+        "\n",
+        "# There are a lot of logs, so it might be worth redirecting to a file so we can see the output easier just use this command\n",
+        "# !nat eval --config_file \"{eval_config_path}\" > output.txt 2>&1 "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "! pip show aiq-aira --version"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Optional: Let's get an CUSTOM EVALUATOR set up together!\n",
```

Comment:
As discussed async, make this section a guide rather than executable, adding a lot of complexity in the notebook and things can break easily, messing up the current package installation